UPGRADING.md

```diff
@@ -1,6 +1,31 @@
 Upgrading Grape
 ===============
 
+
+### Upgrading to >= 1.5.3
+
+### Nil value and coercion
+
+A parameter with a value set to `nil` call custom coercion method, but the coercion is not triggered for missing parameters.
+
+```ruby
+class Api < Grape::API
+  params do
+    optional :value, type: Integer, coerce_with: ->(val) { val || 0 }
+  end
+
+  get 'example' do
+     params[:my_param]
+  end
+  get '/example', params: { value: nil }
+  # 1.5.2 = nil
+  # 1.5.3 = 0
+  get '/example', params: {}
+  # 1.5.2 = nil
+  # 1.5.3 = nil
+end
+```
+
```

Comment:
Add `See [#2164](https://github.com/ruby-grape/grape/pull/2164) for more information.`